candidates.html

```diff
@@ -224,14 +224,16 @@ <h3 class="mb-1 h-underline h-pink h-underline-animate" id="tian-carrie-wilson-f
                   <img class="rounded w-100" title="Owen Miller for Wills" src="https://assets.nationbuilder.com/futureparty/pages/4216/attachments/original/1724677087/g_20240803_142539_cropped_2.png?1724677087" alt="Owen Miller">
                 </a>
             </div>
-            <div class="col-6 col-md-12 order-5 order-md-1 my-md-2">
+            <div class="col-6 col-md-12 order-5 order-md-1 my-md-2 badge-row">
                 <a href="/owen_miller_wills" class="uncolored">
                     <h3 class="mb-1 h-underline h-pink h-underline-animate" id="owen-miller-for-wills">
                       Owen Miller for Wills
                     </h3>
                 </a>
-
-                <span class="badge badge-secondary mt-2 ml-md-3">vic</span>
+                <div>
+                    <span class="badge badge-secondary ml-md-3">vic</span>
```

Comment:
Shouldn't this be a variable eg {{% assign state = "blah" %}}

Reply:
This repository isn't using NationBuilder :slightly_smiling_face: